### PR TITLE
Fix issues with action

### PR DESCRIPTION
**Pull request by DanTheMan827**
This PR updates the funding key to be an array, and also fixes the encoding of the repo blacklist file.

### CODE REVIEW

**Comment by DanTheMan827** [comment]
@Fernthedev

File: src/main.ts [hunk 1]
```diff
@@ -163,7 +163,7 @@ export async function run(): Promise<void> {
       .trim()
       .split('\n')
       .map(line => line.trim())
-    const repoName = `${forkedModRepo.owner.login}/${forkedModRepo.name}`
+    const repoName = `${github.context.repo.owner}/${github.context.repo.repo}`
```
Comment (Fernthedev) [inline src/main.ts:166]: Why was this changed?

Comment (DanTheMan827) [reply src/main.ts:166]: It was wrong.  I wanted the context, not the target repo.

repo name is used for the blacklist
